src/ovh-ui-angular/controllers/countries.json

```diff
@@ -0,0 +1,245 @@
+[
```

Comment:
(optional) The ovh-ui-angular spec data file could also been reused, but it also make sense to not be impacted if the file content changed.